src/systemctl/systemctl.c

```diff
@@ -3291,7 +3308,7 @@ static int check_unit_generic(int code, const UnitActiveState good_states[], int
 static int check_unit_active(int argc, char *argv[], void *userdata) {
         const UnitActiveState states[] = { UNIT_ACTIVE, UNIT_RELOADING };
         /* According to LSB: 3, "program is not running" */
-        return check_unit_generic(3, states, ELEMENTSOF(states), strv_skip(argv, 1));
+        return check_unit_generic(PROGRAM_NOT_RUNNING, states, ELEMENTSOF(states), strv_skip(argv, 1));
```

Comment:
i'd claim we should return either EXIT_NOTRUNNING (7) here actually, or EXIT_FAILURE...